src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1470,4 +1470,22 @@ ModuleVersion : Version of module to import. If used, ModuleName must represent
   <data name="ImplicitRemotingPipelineBatchingNotSupported" xml:space="preserve">
     <value>Command pipeline not supported for implicit remoting batching.</value>
   </data>
+  <data name="ImplicitRemotingPipelineBatchingNotASimplePipeline" xml:space="preserve">
+    <value>Command is not a simple pipeline and cannot be batched.</value>
+  </data>
+  <data name="ImplicitRemotingPipelineBatchingNotImplicitCommand" xml:space="preserve">
+    <value>The pipeline command, {0}, is not an implicit remoting command or approved batching command.</value>
+  </data>
+  <data name="ImplicitRemotingPipelineBatchingWrongSession" xml:space="preserve">
+    <value>The pipeline command, {0}, is for a different remote session and cannot be batched.</value>
+  </data>
+  <data name="ImplicitRemotingPipelineBatchingNoPSSession" xml:space="preserve">
+    <value>The implicit remoting PSSession for batching could not be retrieved.</value>
+  </data>
+  <data name="ImplicitRemotingPipelineBatchingException" xml:space="preserve">
+    <value>Exception while checking the command for implicit remoting batching: {0}</value>
+  </data>
+  <data name="ImplicitRemotingPipelineBatchingSuccess" xml:space="preserve">
+    <value>Implicit remoting command pipeline has been batched for execution on remote target.</value>
```

Comment:
I intentionally left "successfully" out because that implies the batched command ran successfully.  But it really just passed all of the checks.  Running it may result in error.